src/views/projectExplorer/index.ts

```diff
@@ -82,15 +82,19 @@ export default class ProjectExplorer implements TreeDataProvider<ProjectExplorer
           await commands.executeCommand(`code-for-ibmi.setDeployLocation`, undefined, element.workspaceFolder, `${element.description}`);
         }
       }),
-      commands.registerCommand(`vscode-ibmi-projectexplorer.projectExplorer.addLibraryListEntry`, async (element: LibraryList) => {
+      commands.registerCommand(`vscode-ibmi-projectexplorer.projectExplorer.addLibraryListEntry`, async (element: LibraryList | any) => {
         if (element) {
-          const iProject = ProjectManager.get(element.workspaceFolder);
+          const iProject = ProjectManager.getActiveProject();
 
           if (iProject) {
-            const library = await window.showInputBox({
-              prompt: l10n.t('Enter library name'),
-              placeHolder: l10n.t('Library name')
-            });
+            let library = element.name;
+
+            if (element instanceof LibraryList) {
+                library = await window.showInputBox({
+                prompt: l10n.t('Enter library name'),
+                placeHolder: l10n.t('Library name')
+              });
+            }
```

Comment:
Will these actions show if there is no active project?  If so it would be confusing which LIBL is being affected.
Simplest to only show actions if there is an active project and Code 4 i's library list is hidden